packages/trader/src/App/Containers/CashierModal/AccountTransfer/account-transfer-receipt.jsx

```diff
@@ -0,0 +1,92 @@
+import classNames     from 'classnames';
+import PropTypes      from 'prop-types';
+import React          from 'react';
+import { withRouter } from 'react-router';
+import { Button }     from 'deriv-components';
+import Localize       from 'App/Components/Elements/localize.jsx';
+import { localize }   from 'App/i18n';
+import Icon           from 'Assets/icon.jsx';
+import { connect }    from 'Stores/connect';
+
+class AccountTransferReceipt extends React.Component {
+    componentWillUnmount() {
+        this.props.resetAccountTransfer();
+    }
+
+    render() {
+        const {
+            receipt,
+            selected_from,
+            selected_to,
+        } = this.props;
+
+        return (
+            <div className='cashier__wrapper account-transfer__receipt'>
+                <Icon icon='IconTransferDone' className='account-transfer__receipt-icon' />
+                <h2 className='cashier__header'>
+                    <Localize i18n_default_text='Your funds have been transferred.' />
+                </h2>
+                <div className='cashier__transferred-amount cashier__text--bold'>
+                    <span
+                        className={classNames('symbols', `symbols--${selected_from.currency.toLowerCase()}`)}
+                    />
+                    {receipt.amount_transferred}
+                </div>
+                <div className='cashier__transferred-details-wrapper'>
+                    <span className='account-transfer__transfer-details-from'>
+                        <Icon
+                            icon='IconAccountsCurrency'
+                            type={selected_from.mt_icon ?
+                                selected_from.mt_icon.value : selected_from.currency.toLowerCase()}
+                            height={16}
+                            width={16}
+                            vb_height={selected_from.mt_icon ? selected_from.mt_icon.vb_height : undefined}
+                            vb_width={selected_from.mt_icon ? selected_from.mt_icon.vb_width : undefined}
+                        />
+                        <span className='cashier__transferred-details'>
+                            <span className='cashier__text--bold'>{selected_from.text}</span>
+                        </span>
+                    </span>
+                    <Icon className='cashier__transferred-icon' icon='IconBack' />
+                    <span className='account-transfer__transfer-details-to'>
+                        <Icon
+                            icon='IconAccountsCurrency'
+                            type={selected_to.mt_icon ? selected_to.mt_icon.value : selected_to.currency.toLowerCase()}
```

Comment:
Wouldn't it be better if the `toLowerCase()` is done inside the `Icon` component so that the usage is case-insensitive?

Reply:
I don't think it's a good idea to pass in something that you expect to be reflected and suddenly it's changed to something else.